system/logaccess/Grafana/CurlClient/GrafanaCurlClient.cpp

```diff
@@ -1086,10 +1086,10 @@ void GrafanaLogAccessCurlClient::healthReport(LogAccessHealthReportOptions optio
     catch(...)
     {
         status.appendJSONListMessage("Encountered unexpected exception during health report");
-        status = LOGACCESS_STATUS_fail;
```

Comment:
this is outside scope, but caught while updating status variable return. 
This change is necessary and I hope we can include it here.